gips/data/sentinel2/sentinel2.py

```diff
@@ -248,7 +248,7 @@ class sentinel2Asset(gips.data.core.CloudCoverAsset,
     }
 
     # default resultant resolution for resampling during to Data().copy()
-    _defaultresolution = (10, 10)
```

Comment:
There's references to `_defaultresolution` in a few other data sources (cdl, weld, landsat, prism, daymet, sarannual, sar, modis, merra), but the only usage it gets is in the daymet driver to help define a geotransform and in the `Data().copy()` method. I agree it probably isn't needed since I can't find a usage of the `Data().copy()`, but it feels safer to keep this here until when/if we retire `Data().copy()`?